external/vcm/vcm/__init__.py

```diff
@@ -25,6 +25,8 @@
     potential_temperature,
     pressure_at_interface,
     surface_pressure_from_delp,
+    column_integrated_heat,
```

Comment:
This would be a good opportunity to use a clearer name than "heat". E.g. `column_enthalpy` or `column_internal_energy`.